spec/features/posts_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'show page', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Morris', bio: 'This is the bio',
+                        photo: 'https://i.im.ge/2022/09/22/1Ud6MY.druid.th.png')
+    @id = @user.id
+  end
+  describe 'test' do
+    it 'shows user name' do
+      visit "users/#{@id}/posts"
+      expect(page).to have_content(@user.name)
+    end
+
+    it 'shows the posts counter' do
+      visit "users/#{@id}/posts"
+      expect(page).to have_content(@user.posts_counter)
+    end
+
+    it 'shows profile picture' do
+      visit "users/#{@id}/posts"
+      expect(page).to have_css("img[src='#{@user.photo}']")
+    end
+
+    describe 'create a post' do
+      before(:each) do
+        @post = Post.create(title: 'My first post', author: @user, text: 'Text of my first post')
+        @post_id = @post.id
+      end
+
+      it 'shows string posts' do
+        visit "users/#{@id}/posts"
+        expect(page).to have_selector('h1', text: 'Posts')
+      end
+
+      it 'show a post title' do
+        visit "users/#{@id}/posts"
+        expect(page).to have_content(@post.title)
+      end
+
+      it 'shows the amounts of comments' do
+        visit "users/#{@id}/posts"
+        expect(page).to have_content(@post.comments_counter)
+      end
+
```

Comment:
- [ ] It would be better if all our test cases mark green. Currently, we have a few pending test cases we need to resolve :100: Let us kindly address it
![Screenshot from 2022-09-28 00-18-06](https://user-images.githubusercontent.com/46388113/192660164-c0d821bd-19c4-4cc5-b150-6f7087e1e12b.png)

Reply:
they are not test files.